tests/providerTest.js

```diff
@@ -35,6 +36,25 @@ describe('provider testing', () => {
     }
 
     opts.stateHandlers = {
+      'user exists': (setup, params) => {
+        if (setup) {
+          fetch(process.env.PROVIDER_BASE_URL + '/ocs/v1.php/cloud/users', {
+            method: 'POST',
+            body: `userid=${params.userName}&password=${params.password}`,
```

Comment:
I think you can send simple json body here, no need to complicate this one

Reply:
do you mean

Reply:
I cannot create user with json format
```json
...
body: { "userid": "user1", "password": "1234" }
headers: {
  <validauth>,
  "Content-Type": "application/json"
}
...
```

```xml
<ocs>
    <meta>
        <status>failure</status>
        <statuscode>101</statuscode>
        <message>A valid username must be provided</message>
    </meta>
    <data/>
</ocs>
```